app/ImageConversion.hs

```diff
@@ -1,8 +1,81 @@
 module ImageConversion
-    (createImageFile, removeImage) where
+    (getGraphImage, getTimetableImage, createPDF, createImageFile, removeImage) where
 
 import System.Process
 import GHC.IO.Handle.Types
+import TimetableImageCreator (renderTable)
+import qualified Data.Map as M
+import System.Random
+import Svg.Generator
+import Database.Tables (GraphId)
+
+-- | Creates an image, and returns the name of the svg used to create the
```

Comment:
You should move these to a separate file, since they are helper functions that are shared between multiple actions.